scripts/Engineering/EnggUtils.py

```diff
@@ -426,6 +436,35 @@ def replace_patterns(line, patterns, replacements):
                         ("INS    INCBM  ob+mon_{0}_North_and_South_banks.his".
                          format(ceria_run)).ljust(80) + '\n']
 
+        if bk2bk_params:  # template params not overwritten if none provided
+
+            patterns += ["INS  {}PRCF11".format(b_idx + 1), "INS  {}PRCF12".format(b_idx + 1)]
+
+            if len(gamma_0_gamma_1) == 0:  # set gamma_0 and gamma_1 if not properly read from the parameter file
+                if _bank_name == "North":
+                    gamma_0_gamma_1_val = "0.000000E+00   0.514723E+01 "
+                elif _bank_name == "South":
+                    gamma_0_gamma_1_val = "0.000000E+00   0.652544E+01 "
+                else:
+                    gamma_0_gamma_1_val = "0.000000E+00   0.000000E+00 "
+            else:  # if correctly read from parameter file
+                gamma_0_gamma_1_val = gamma_0_gamma_1[b_idx]
+
+            replacements += [("INS  {0}PRCF11   {1:.6E}   {2:.6E}   {3:.6E}   {4:.6E}".
```

Comment:
Could use f-strings here but I do like that spreading the format over several lines allows you to put a comment specify which parameter is which so I'll leave it up to you.

Reply:
Yea, PyCharm suggested changing to fstrings but when I did, it looked a little less clear and having the comments here works nicely. It also matches the other replacements lines above.